### PR TITLE
[Practice-4]: section-management-item

### PR DESCRIPTION
**Pull request by minhtringuyen9623**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -61,6 +62,133 @@ <h1 class="banner-title">Bilgi Teknolojilerinde 23 Yıllık Tecrübe</h1>
...
+        <div class="management-item-category-list">
+          <!-- item-1 -->
+          <div class="category management-item-category">
+            <div class="category-frame">
+              <div class="category-frame-img">
+                <img
+                  src="./assets/icons/analysis-icon.svg"
+                  alt="analysis-icon"
+                />
+                <div class="circle-icon"></div>
```
Comment (toannguyen1710mt) [inline src/index.html]: use span element instead.

File: src/index.html [hunk 2]
```diff
@@ -61,6 +62,133 @@ <h1 class="banner-title">Bilgi Teknolojilerinde 23 Yıllık Tecrübe</h1>
           </form>
         </div>
       </section>
+
+      <!-- section-management-item -->
```
Comment (toannguyen1710mt) [inline src/index.html]: redefine to section management is enough.

File: src/index.html [hunk 3]
```diff
@@ -61,6 +62,133 @@ <h1 class="banner-title">Bilgi Teknolojilerinde 23 Yıllık Tecrübe</h1>
...
+      <section class="management-item container">
```
Comment (toannguyen1710mt) [inline src/index.html]: Rename to management

File: src/index.html [hunk 4]
```diff
@@ -61,6 +62,133 @@ <h1 class="banner-title">Bilgi Teknolojilerinde 23 Yıllık Tecrübe</h1>
...
+        <div class="heading-content flex-center">
+          <h2 class="management-item-heading">Kalite ve Süreç Yönetimi</h2>
```
Comment (toannguyen1710mt) [inline src/index.html]: rename to management-heading

File: src/index.html [hunk 5]
```diff
@@ -61,6 +62,133 @@ <h1 class="banner-title">Bilgi Teknolojilerinde 23 Yıllık Tecrübe</h1>
...
+          <p class="management-item-decs">
```
Comment (toannguyen1710mt) [inline src/index.html]: rename to management-description. Shouldn't be abbreviated

File: src/index.html [hunk 6]
```diff
@@ -61,6 +62,133 @@ <h1 class="banner-title">Bilgi Teknolojilerinde 23 Yıllık Tecrübe</h1>
...
+            Müşterilerimizin yüksek kalite seviyelerini koruyabilmeleri için
+            farklı sektörlerde tecrübe kazanmış uzman kadrolarımızla Proje
+            Yönetimi, İş Analizi ve Test Yönetimi hizmetleri sunmaktayız.
+          </p>
+        </div>
+        <div class="management-item-category-list">
```
Comment (toannguyen1710mt) [inline src/index.html]: also, remove `item` please

File: src/index.html [hunk 7]
```diff
@@ -61,6 +62,133 @@ <h1 class="banner-title">Bilgi Teknolojilerinde 23 Yıllık Tecrübe</h1>
...
+          <!-- item-1 -->
+          <div class="category management-item-category">
+            <div class="category-frame">
```
Comment (toannguyen1710mt) [inline src/index.html:77]: Why do you need wrapper here?

File: src/index.html [hunk 8]
```diff
@@ -61,6 +62,133 @@ <h1 class="banner-title">Bilgi Teknolojilerinde 23 Yıllık Tecrübe</h1>
...
+              <div class="category-frame-img">
+                <img
+                  src="./assets/icons/analysis-icon.svg"
+                  alt="analysis-icon"
+                />
+                <div class="circle-icon"></div>
+              </div>
+            </div>
+            <div class="category-content">
```
Comment (toannguyen1710mt) [inline src/index.html:86]: Why do you need wrapper here?

File: src/style/components/category.css [hunk 9]
```diff
@@ -0,0 +1,42 @@
+.category {
```
Comment (toannguyen1710mt) [inline src/style/components/category.css]: Please make a global class `column-center` to reuse for many where.

File: src/style/sections/management-item.css [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+.management-item-category-list {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 80px;
+  margin-top: 40px;
+}
+
+.management-item-category:nth-child(2) .circle-icon {
```
Comment (toannguyen1710mt) [inline src/style/sections/management-item.css]: Is it still right when the item change the order in different devices (laptop, tablet)?

**Review by toannguyenhuu-agilityio — APPROVED** [review]

**Pull request merged**